chain/ethereum.ts

```diff
@@ -27,18 +27,23 @@ export namespace ethereum {
    *
    * Big enough to fit any pointer or native `this.data`.
    */
-  export type ValuePayload = u64
+  export type ValuePayload = usize
```

Comment:
We'll probably have to keep this as `u64` because of how we structure the layout on `graph-node`.

One thing I did on that project's tests was to keep the `Payload` as `u64` and do the `changetype` castings as `u32`, the addresses/pointers will fit normally inside this type 🙂 

Just a bit more explanation, on `graph-node` we use the same structure for both pointers (eg: Payload) and primitive values (eg: numbers, enums and booleans). And this structure is a `u64` there.

Reply:
You're right! In AssemblyScript side, I think any pointer should fit in `isize/usize`, however, even if `usize` should be equals to `u64` in WASM64, it makes totally sense to keep as `u64` to be explicit and compatible with WASM32.